app/src/main/AndroidManifest.xml

```diff
@@ -10,15 +10,28 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
         <activity
-            android:name=".MainActivity"
-            android:label="@string/app_name"
+
+            android:name=".WelcomePage"
+            android:label="@string/title_activity_welcome_page"
```

Comment:
While this changes the Welcome Page title, it also changes the title of the app itself (the app is now called "Welcome Page") - this should probably be changed to string/app_name so it says "Access Collective"
The easiest way to change the title of an Activity is to call setTitle(getString(R.string.title_activity_welcome_page)) in the onCreate method of WelcomePage.java